### ISSUE TITLE
Textfield labels receive VoiceOver (iOS) focus separately from actual Textfields they are associated with

### ISSUE THREAD

**Issue by nairmanu** [OfficeDev/office-ui-fabric-react#12233]
### Environment Information

- **Package version(s)**: 7.83.1
- **Browser and OS versions**: Safari, iPhone 11, iOS 13.3.1

### Describe the issue:
Textfield labels receive VoiceOver (iOS) focus separately from actual Textfields they are associated with when the field is marked as required. If it is not set as required, the control behaves as expected.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/manunair3/pen/NWqwjjv

#### Actual behavior:
Textfield label receive focus separately from actual Textfields. This results in the label getting announced twice. 

#### Expected behavior:
Textfield labels shouldn't get separate focus.

**Comment by aneeshack4** [comment]
@nairmanu thanks for reaching out to us! I don't repro this on iOS - it only announces the label once. I checked Narrator as well - also announces only once. Are you sure you're using the latest version of Fabric and iOS and whichever browser you're using?

**Comment by nairmanu** [comment]
Sorry I wasn't clear earlier. When I said label getting announced twice, I meant the label gets a separate focus from actual textfield. As a result the focus first reaches the label, VoiceOver announces the label. Then on swipe right, the focus reaches actual Textfield and VoiceOver announces the aria-label of the Textfield (which is usually the same as the label itself), says it is a TextField and says it is a required field and that the user can double tap to edit. 

For e.g. if the TextField is having label "Sample", VoiceOver will announce the following.
1) Focus reaches label, VoiceOver announces "Sample"
2) On swipe right, focus reaches Textfield, VoiceOver announces "Sample *, Textfield, required, double tap to edit".

Expected behavior is that label shouldn't get VoiceOver focus. 

Sample code:
`<TextField label="Sample" required />`

**Comment by aneeshack4** [comment]
Ah I see what you mean - I'm able to repro this now on Safari/iOS: 
![image](https://user-images.githubusercontent.com/4161791/76370665-a1471d00-6305-11ea-8085-47f05059df0b.png)
I'm having a lot of trouble with VoiceOver on Mac so I don't know if this repros on Mac too. But it doesn't repro with Narrator on Windows and as far as I can tell, we're following the ARIA spec for Textfield. So I think this is a VoiceOver/Apple bug and not something we can fix on our end - unless you see something in the Fabric TextField that's missing in terms of a11y.
